### ISSUE TITLE
Dynamically Adding and modifying events

### ISSUE THREAD

**Issue by alberttwong** [OSAS/rh-events#135]
Is there a way to do this?   I know we have a bunch of red hat speakers in our webinars (Nick Mann) and in our meetups (Crystal Murphy).

**Comment by mscherer** [comment]
Could you clarify what you mean, I am not sure to understand the question ?

**Comment by alberttwong** [comment]
What I'm looking for is a way to insert/add more RH speaker events into your feed automatically/programmatically.  Reading the instructions, it seems like the only option is to edit the raw JSON file.

**Comment by mscherer** [comment]
Yep, and yaml can be programatically updated. And you can use the github API and git to script the patch submission part, so while it is not turn key at the moment, it doesn't seems impossible to automate. The only issue is merging the PR, and that's more a policy decision than a technical issue.

**Comment by garrett** [comment]
I've added some scripts I wrote a while back that automates scraping of Lanyrd and iCalendar files. 

The `.ics` script is really rough; it might not work as intended, and the Lanyrd script did work well at one point; I haven't tested it lately, however. Both export YAML data which can be redirected to a file or copy/pasted. The resulting YAML should be correct, but you might want to look it over and make minor modifications before submitting a pull request.

https://github.com/OSAS/event-tools

It's all in Ruby, but should be pretty straightforward.
